config/routes.rb

```diff
@@ -1,12 +1,14 @@
 Rails.application.routes.draw do
 
   devise_for :admins
-  devise_for :volunteers, :controllers => { :registrations => "registrations" }
+  devise_for :volunteers, :controllers => { :registrations => "my/registrations" }
```

Comment:
this should not be changed to `/my/registration`. this will make `http://localhost:3000/volunteers/sign_up` inaccessible. It is raising an exception `uninitialized constant My::RegistrationsController` because the route cannot find the controller (which is in `RegistrationsController`)